### ISSUE TITLE
Unable to install V2 of this package with Filament  3.x

### ISSUE THREAD

**Issue by viraljetani** [Z3d0X/filament-fabricator#86]
Tried installing the package but I am getting a sub-dependency error about pboivin/filament-peek ^2.0 

I am on Laravel ^10.10, Filament ^3.0-stable

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires z3d0x/filament-fabricator ^2.0 -> satisfiable by z3d0x/filament-fabricator[v2.0.0, v2.0.1].
    - z3d0x/filament-fabricator[v2.0.0, ..., v2.0.1] require pboivin/filament-peek ^2.0 -> found pboivin/filament-peek[v2.0.0-alpha1, ..., 2.x-dev] but it does not match your minimum-stability.
```

**Comment by viraljetani** [comment]
For a workaround I installed the package - pboivin/filament-peek:"^2.0.0-beta4" before installing this package and I was able to install it successfully.

**Comment by rolfkempf** [comment]
The workaround is fine but I think the issue should stay open. The missing dependancy could prevent users from trying this package because the installation doesn´t work as expected.

**Comment by Z3d0X** [comment]
fixed by https://github.com/Z3d0X/filament-fabricator/commit/771de45e59a6fba07ba9d42e1492beeef003a3bc

**Comment by pboivin** [comment]
Hey folks, I just released [Peek 2.0](https://github.com/pboivin/filament-peek/releases/tag/v2.0.0) which should avoid the confusion. There are still minor issues related to Filament v3 but, to the best of my knowledge, nothing that should impact the plugin in the context of Fabricator.
